.github/workflows/main.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - run: yarn
-      - run: yarn lint --no-fix
+      - run: yarn lint --no-fix | grep "error:"
```

Comment:
I don't really like grepping this output as you miss the body of the error, so I opened https://github.com/vuejs/vue-cli/issues/6950